src/nft/components/explore/Table.tsx

```diff
@@ -148,8 +148,8 @@ export function Table<D extends Record<string, unknown>>({
     <GlowEffect>
       <table {...getTableProps()} className={styles.table}>
         <thead className={styles.thead}>
-          {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()} key={headerGroup.id}>
+          {headerGroups.map((headerGroup, index) => (
+            <tr {...headerGroup.getHeaderGroupProps()} key={index}>
```

Comment:
for arrays like `headerGroups` that can dynamically change in size - don't use `index` as a key because it's not guaranteed that the same instance will have the same index

can you investigate why `headerGroup.id` is not working instead?